src/Database/Statement/PDOStatement.php

```diff
@@ -90,13 +90,13 @@ public function bindValue($column, $value, $type = 'string')
      */
     public function fetch($type = 'num')
```

Comment:
You could then also change this magic string.

Reply:
Wonder if a change to the statement directory where these strings are used is called for in this case.